tOpt/SDFANIMOptimizer.py

```diff
@@ -11,7 +11,11 @@
 
 '''
 
-import pyNeuroChem as neuro
+try:
+    import pyNeuroChem as neuro
+except ImportError:
+    pass
```

Comment:
we should add a warning message here.
An alternative would be to implement a completely separate sdfTorchAniMOptimizer class